lib/DDG/Spice/Syntax.pm

```diff
@@ -0,0 +1,30 @@
+package DDG::Spice::Syntax;
+
+# ABSTRACT: Look up syntax for a programming language
+
+# Start at http://docs.duckduckhack.com/walkthroughs/forum-lookup.html if you are new
```

Comment:
Can you bomb out these comments? They're only really for instruction (with the exclusion of the # ABSTRACT)